src/Behavioral.Automation/FluentAssertions/Abstractions/IAssertionBuilder.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Behavioral.Automation.Elements;
+using Behavioral.Automation.Model;
+using System;
+
+namespace Behavioral.Automation.FluentAssertions.Abstractions
+{
+    public interface IAssertionBuilder
+    {
+        IAssertionBuilder And { get; }
+        IAssertionBuilderWithInversion Not { get; }
```

Comment:
this may lead to incorrect syntax like 
`X.Should(....).Not.And.Not.And.....`
this case may be resolved if 'Not' will return some different interface with narrow list of properties & methods. Same for 'Not'

Reply:
Done. And removed from root interface and new interface implemented